assets/scss/main.scss

```diff
@@ -6,73 +6,73 @@
 @import "settings/fonts";
 
 //Adding styling to components such as toolbar, dialogue box, etc.
-@mixin component($conf: ()) {
+@mixin component($config: ()) {
 
 	// Dumping CSS
-	.#{map-get($conf, name)} {
-		width: map-get($conf, size);
-		$theme: map-get($conf, theme);
+	.#{map-get($config, name)} {
+		width: map-get($config, size);
+		$theme: map-get($config, theme);
 		background: map-get($bg-clr, $theme) !important;
 		color: map-get($txt-clr, $theme);
-		font-family: map-get($conf, font-family);
+		font-family: map-get($config, font-family);
 	}
 }
 
 //Adding styling to text elements
-@mixin text($conf: ()) {
+@mixin text($config: ()) {
 
 	// Dumping CSS
-	.#{map-get($conf, name)} {
-		width: map-get($conf, size);
-		$border: map-get($conf, border);
-		$theme: map-get($conf, theme);
+	.#{map-get($config, name)} {
+		width: map-get($config, size);
+		$border: map-get($config, border);
+		$theme: map-get($config, theme);
 		color: map-get($txt-clr, $theme) !important;
 		border-color: map-get($txt-clr, $border) !important;
-		font-family: map-get($conf, font-family);
+		font-family: map-get($config, font-family);
 	}
 }
 
 $component-toolbar: (
 	theme: map-get($config, theme),
 	name: 'md-toolbar-tools',
-	font-family: map-get($conf, font-family),
+	font-family: map-get($config, font-family),
```

Comment:
This change and corresponding changes in other component attributes are enough. You don't need to make change in mixin-component and mixin-text. That is not a typo. It is a parameter which receives the value of `$config`. So, its perfectly fine if it has some other name.